dnsapi/README.md

```diff
@@ -1259,6 +1259,22 @@ acme.sh --issue --dns dns_online -d example.com -d www.example.com
 
 `ONLINE_API_KEY` will be saved in `~/.acme.sh/account.conf` and will be reused when needed.
 
+## 66. Use UltraDNS API
+
```

Comment:
can you please share the ultradns website url here ?

Reply:
@Neilpang - I've added the URL for the UltraDNS portal in the dnsapi/README.md file.